Documentation/development-flow.md

```diff
@@ -33,11 +33,41 @@ cd $GOPATH/src/github.com/rook
 
 # Clone your fork, where <user> is your GitHub account name
 git clone https://github.com/<user>/rook.git
-
 cd rook
+```
+
+### Build
 
-# build rook
+```bash
+# build all rook storage providers
 make
+
+# build a single storage provider, where the IMAGES can be a subdirectory of the "images" folder:
+# "cassandra", "ceph", "cockroachdb", "edgefs", "minio", or "nfs"
+make IMAGES="cassandra" build
+
+# multiple storage providers can also be built
+make IMAGES="cassandra ceph" build
+```
+
+### Development Settings
+
+To provide consistent whitespace and other formatting in your `go` and other source files, it is recommended you apply
+the following settings in your IDE:
+- Format with the `goreturns` tool
+- Trim trailing whitespace
+
+For example, in VS Code this translates to the following settings:
+
+```json
+{
+    "editor.formatOnSave": true,
+    "go.buildOnSave": true,
```

Comment:
mine doesn't list `true` as an option, using VS Code 1.30.1.  I wonder if that's a stale comment that's been in the settings file for a long time?
```
 // Compiles code on file save using 'go build -i' or 'go test -c -i'. Options are 'workspace', 'package or 'off'.
  "go.buildOnSave": "package"
```

Reply:
I would suggest `"package"` as the setting here. Rebuilding the entirety of Rook on save is resource intensive. Though the recommendations could be `"go.buildOnSave": "package" // or "workspace" if your machine builds Rook quickly

Reply:
Yep, must be a very old setting. Agreed that "package" works well.